### ISSUE TITLE
Desklet Feature Request

### ISSUE THREAD

**Issue by rcalixte** [linuxmint/cinnamon-spices-desklets#1086]
### Desklet name and maintainer

battery@schorschii @schorschii

### What would you like to see?

It would be great to have an option to display a battery percentage and/or time remaining to charge/discharge. This could be accomplished by changing the "Show battery in percent" option to a new "Battery display" dropdown (while migrating the existing user preference so that the upgrade is transparent). I was able to implement a similar duration option recently for the batterymonitor@pdcurtis applet and I think it would be a good addition here as well. There is also now an example of settings migration in the bbcwx@oak-wood.co.uk desklet as that was done recently too.

If the idea is amenable, I have no problem taking the time to implement it or you can do it at your own leisure. Thanks again for an awesome desklet!

**Comment by schorschii** [comment]
Yes, it would be nice to have this information in my desklet too. I would implement this with an additional smaller label below the capacity label and with an additional switch in the settings "Show remaining time" (I think this allows more flexible configuration than a dropdown - so users can freely combine the options to display icon, capacity and remaining time).

I already created a design draft in #1087. All that's missing is actually reading out the remaining time. Feel free to add commits on my branch.
